test/jasmine/tests/toimage_test.js

```diff
@@ -0,0 +1,112 @@
+// move toimage to plot_api_test.js
+//  once established and confirmed?
+
+var Plotly = require('@lib/index');
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var subplotMock = require('../../image/mocks/multiple_subplots.json');
```

Comment:
same here: `@mocks/multiple_subplots.json` instead of `../../image/mocks/multiple_subplots.json`.